devices/neo/NAS-WR01B_TS011F.json

```diff
@@ -1,11 +1,20 @@
 {
   "schema": "devcap1.schema.json",
-  "manufacturername": [ "_TZ3000_gjnozsaz", "_TZ3000_zloso4jk" ],
-  "modelid": [ "TS011F", "TS011F" ],
-  "product": "NEO Smart Power Plug 16A",
+  "manufacturername": [
+    "_TZ3000_gjnozsaz",
+    "_TZ3000_w0qqde0g",
+    "_TZ3000_zloso4jk",
+    "_TZ3000_rdfh8cfs"
+  ],
+  "modelid": [
+    "TS011F",
+    "TS011F",
+    "TS011F",
+    "TS011F"
+  ],
+  "product": "NAS-WR01B",
```

Comment:
Could you please add what kind of device this is? Just adding smart plug should suffice.